### PR TITLE
Course details

### PR DESCRIPTION
**Pull request by nkosi-tauro**
Added the Course Details Screen

image

### CODE REVIEW

File: app/src/main/res/layout/activity_course_details.xml [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".CourseDetails">
+
+
+    <TextView
+        android:id="@+id/textView2"
```
Comment (redatawfik) [inline app/src/main/res/layout/activity_course_details.xml:11]: Please add a meaningful id

File: app/src/main/res/layout/activity_course_details.xml [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="17dp"
+        android:layout_marginEnd="134dp"
+        android:text="End Date"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/textField" />
+
+    <com.google.android.material.textfield.TextInputLayout
+        android:id="@+id/textField"
```
Comment (redatawfik) [inline app/src/main/res/layout/activity_course_details.xml:21]: Please add a meaningful id

File: app/src/main/res/layout/activity_course_details.xml [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+
+        <com.google.android.material.textfield.TextInputEditText
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:hint="Course Title" />
+
+    </com.google.android.material.textfield.TextInputLayout>
+
+    <Button
+        android:id="@+id/button3"
```
Comment (redatawfik) [inline app/src/main/res/layout/activity_course_details.xml]: Same

File: app/src/main/res/layout/activity_course_details.xml [hunk 4]
```diff
@@ -0,0 +1,156 @@
...
+        android:layout_marginBottom="16dp"
+        android:backgroundTint="#DCDDDD"
+        android:text="CANCEL"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <Button
+        android:id="@+id/button2"
```
Comment (redatawfik) [inline app/src/main/res/layout/activity_course_details.xml]: Same

File: app/src/main/res/layout/activity_course_details.xml (hunk 1 above)
Comment (redatawfik) [inline app/src/main/res/layout/activity_course_details.xml]: Please add a meaningful id

File: app/src/main/res/layout/activity_course_details.xml (hunk 2 above)
Comment (redatawfik) [inline app/src/main/res/layout/activity_course_details.xml]: Same

**Review by redatawfik — COMMENTED** [review]
Thanks, Looks good to me + some comments please address them.

File: .idea/render.experimental.xml [hunk 5]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="RenderSettings">
+    <option name="showDecorations" value="true" />
+  </component>
+</project>
```
Comment (redatawfik) [inline .idea/render.experimental.xml:6]: What is this change??

File: app/src/main/res/layout/activity_course_details.xml (hunk 3 above)
Comment (nkosi-tauro) [reply app/src/main/res/layout/activity_course_details.xml]: Changes made

File: app/src/main/res/layout/activity_course_details.xml (hunk 1 above)
Comment (nkosi-tauro) [reply app/src/main/res/layout/activity_course_details.xml]: Changes made

File: .idea/render.experimental.xml (hunk 5 above)
Comment (nkosi-tauro) [reply .idea/render.experimental.xml:6]: Not sure what this file is, I believe it was created automatically.

File: app/src/main/res/layout/activity_course_details.xml (hunk 4 above)
Comment (nkosi-tauro) [reply app/src/main/res/layout/activity_course_details.xml]: Changes made

**Pull request merged**